governance/polls/Proposal - Rates V2 Poll - October 12, 2020.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Rates V2 Poll - October 12, 2020
+summary: Signal your support to execute the Rates V2 plan.
+discussion_link: https://forum.makerdao.com/t/signal-request-rates-v2/4481
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Rates V2 Poll - October 12, 2020
+
+The MakerDAO Governance Facilitator has placed a governance poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Maker Governance community which gives Maker Governance the chance to signal their support for the implementation of the Rates V2 plan.
+
+This non-standard weekly Governance Poll ([FAQ](https://community-development.makerdao.com/makerdao-mcd-faqs/faqs#governance)) will be active for 3 days beginning on Monday, October 12 at 16:00 UTC.
+
+## Review
+
+Please refer to the [Signal Request](https://forum.makerdao.com/t/signal-request-rates-v2/4481) for a detailed outline of the proposal.
+
+Plan:
+
+* Scrap the current Stability Fee formula involving the Base Rate. The Base Rate will no longer be voted on after this proposal passes, and a MIP18 vote will be added to deprecate it formally.
+* Formation of the Rates Working Group. A somewhat loosely defined group consisting of Domain Team Facilitators and community members. Initial members being @hexonaut, @Primoz, @LongForWisdom, @monet-supply, @SebVentures and @Akiva.
+* Introduction of the Competitive Rate: A rate which is based on the competitive landscape. For example if our competition is offering borrowing at 5% for a particular asset, we may decide to offer that same asset for 4%.
+* The Rates Working Group will keep track of governance directives, risk premiums and competitive rates to produce a set of stability fees for each vault type. Mostly this will involve setting the rates by competitive analysis coming from risk.
+* The Risk Team will start producing Risk Premium / Competitive Rates on a monthly basis which get handed off to the Rates Working Group who will propose the Stability Fee rate changes.
+* Dai Savings Rate will remain at 0% until governance signals otherwise.
+* Development of a more formal MIP as this process is smoothed out.
+
+## Next Steps
+
```

Comment:
Maybe need to keep the header line: "If this proposal passes then: "
It probably makes sense to explain the negative too: "If this proposal fails to pass then..."